db/migrate/20240319225935_create_payments.rb

```diff
@@ -0,0 +1,67 @@
+class CreatePayments < ActiveRecord::Migration[7.0]
+  def up
+    create_table :payments, id: :uuid do |t|
+      t.references :crime_application, type: :uuid, foreign_key: true, null: false
+      t.string :type, null: false
+
+      t.string :payment_type, null: false
+      t.integer :amount, null: false
+      t.string :frequency, null: false
+      t.jsonb :metadata, default: {}, null: false
+
+      t.timestamps
+
+      # A crime application can only ever have a single payment_type per payment
+      t.index [:crime_application_id, :type, :payment_type], unique: true, name: "index_payments_crime_application_id_and_payment_type"
+    end
+
+    ## Migrate existing data, avoid ActiveRecord due to STI classes having incorrect superclass
+    [IncomePayment, IncomeBenefit, OutgoingsPayment].each do |klass|
+      sql = <<-SQL
+        INSERT INTO payments (type, crime_application_id, payment_type, amount, frequency, metadata, created_at, updated_at)
```

Comment:
redundant `type`

(Personally I'd be happy if we did not migrate the data. Rehydration from Datastore will still work. So it would only really impact in progress staging apps.)